src/server/HelixImportProject.js

```diff
@@ -28,6 +29,20 @@ export class HelixImportProject extends BaseProject {
     return this._allowInsecure;
   }
 
+  withHeadersFile(value) {
+    this._headersFile = value;
+    this._cliHeaders = (value) ? fs.readJSON(value) : {};
```

Comment:
that doesn't work. need too `await` the async call.

Reply:
Actually, that's a really good point.. why the tests are not capturing this ?

Reply:
With https://github.com/adobe/helix-cli/pull/2368/commits/00503e1e1d11d7fa0fe9e05b4b89c9e1e61fb7de the JSON file is read asynchronously